empathy/README.md

```diff
@@ -0,0 +1,43 @@
+1. Ask What, Why, and How? Ask these questions over and over until you reach the underlying answer.
+2. Use [empathy mapping](https://www.nngroup.com/articles/empathy-mapping/) to consider users in relation what the user:
+    - Says
+    - Thinks
+    - Does
+    - Feels
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/57373296/75280377-4f69a780-57db-11ea-85b6-245c222f8f9b.png" width="250" height="250">
+</p>
+
+<br>
+
+<br>
+Putting people first is the foundation of human-centric design, and it requires a deep understanding of their true needs.  Often, the affected people are unaware of these needs themselves.  This is why it's critically important to employ the strategies above to ensure identification of the root problem.
+<br>
+<br>
+<hr>
+
+**CASE STUDY**
+
+Consider this example scenario: a municipality has an aging infrastructure and an archaic system for managing road crews.  They receive numerous phone calls and verbal complaints reporting road issues. These issues are then recorded in spreadsheets or email threads before a road crew is dispatched.  Many issues are never acted upon and citizens often call with complaints that repairs have not been completed.
+
+The customer requests that a software development team provide a solution that ingests and consolidates data from multiple and varied existing data sources (email, spreadsheets) and then provides a daily report of new issues.
+
+The true solution however - the most meaningful result - might be to provide a free mobile app that allows citizens to easily report a road issue (pothole, debris, etc.) which loads a centralized queue for dispatchers.  The app allows the road crew to report real-time status of repairs.
+
+In this example, the customer's identification of the problem is influenced by the immediate pain point of having too much incoming data in too many formats.  The designer, however, delves to the root problem and envisions a more innovative solution.  One that not only addresses the unmanaged data, but provides real-time feedback to the citizen, dispatcher, and road crew so that nothing falls through the cracks and everyone is accountable for the condition of the roads.
+
+<hr>
+<br>
+
+In [The Design Of Everyday Things](https://jnd.org/the-design-of-everyday-things-revised-and-expanded-edition/) Don Norman states "One of my rules in consulting is simple: never solve the problem I am asked to solve. Why such a counterintuitive rule? Because, invariably, the problem I am asked to solve is not the real, fundamental, root problem.  It is usually a symptom."
+
+Here are some additional resources for further reading on the Empathize phase:
+- [Guide to Building Empathy for your Users](https://careerfoundry.com/en/blog/ux-design/what-is-empathy-in-design-thinking/)
+- [Stanford Design Thinking Process Guide](https://dschool-old.stanford.edu/sandbox/groups/designresources/wiki/36873/attachments/74b3d/ModeGuideBOOTCAMP2010L.pdf)
```

Comment:
Phase one of deign thinking has topmost need to empathize with the users. You can empathize by direct and indirect observation , engagement and doing sample case studies, immersing self in user shoes and their work environment.